elements/pfe-autocomplete/src/pfe-autocomplete.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Autocomplete",
+  "description": "Autocomplete provides options in a dropdown list as user types in an input box by showing result from an api call.",
+  "type": "object",
+  "tag": "pfe-autocomplete",
+  "class": "pfe-autocomplete",
+  "category": "content",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object",
+      "properties": {
+        "content": {
+          "title": "Content",
+          "type": "array",
+          "namedSlot": false,
+          "items": {
+            "oneOf": [
+              {
+                "$ref": "raw"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "attributes": {
+      "title": "Attributes",
+      "type": "object",
+      "properties": {
+        "debounceTimer": {
+          "title": "Debounce",
+          "description": "The amount of time that should pass before the next API call is made",
+          "type": "string",
+          "prefixed": false
+        },
+        "init-value": {
+          "title": "Initial value",
+          "description": "An initial value to show in the input field",
+          "type": "string",
+          "prefixed": false
+        },
+        "is-disabled": {
```

Comment:
Schema uses underscores to separate content in a key rather than camel case or dashes

Reply:
Fixed.